test/e2e/azure_machinepool_drain.go

```diff
@@ -47,7 +47,9 @@ import (
 )
 
 const (
-	AzureMachinePoolDrainSpecName = "azure-mp-drain"
+	AzureMachinePoolDrainSpecName   = "azure-mp-drain"
+	waitForDrainOperationTimeout    = 20 * time.Minute
+	waitForDrainSleepBetweenRetries = 10 * time.Second
```

Comment:
isn't 10 seconds pretty long in between retries if we're concerned about the node changing states too quickly?

Reply:
Fair, and since we're currently just testing for drain start signal, we can move the timeout down to something lower as well